### ISSUE TITLE
[Android] Disable Display Size setting support

### ISSUE THREAD

**Issue by Soap-141** [unoplatform/uno#13598]
### What would you like to be added

Might not be doable, made some research on it but it would be nice if we could ignore the display size setting on Android because it can mess up the UI quit a lot.

I guess the obvious fix would be to code better XAML with less fixes sizes and less range of sizes so there is no clipping and other weird behavior but sometime we have to do some workarounds or have fixed size / range of sizes that cause more issue when playing with this setting.

Default vs Larger with font scaling disabled.

![Capture_105](https://github.com/unoplatform/uno/assets/57812285/484e6b36-3a1b-4c3f-8e95-79e95ed9f6c2)
![Capture_106](https://github.com/unoplatform/uno/assets/57812285/27e07cb4-7916-483c-8ca1-a6d5ba33d585)

### Why is this needed

A flag to disable the display size scaling.

### For which platform

Android

**Comment by jeromelaban** [comment]
Thanks for the report.

There are those two properties that should be having an effect:
https://github.com/unoplatform/uno/blob/669dbc7421905dcd8cb7caf09c56834d35ee24d9/src/Uno.UI/FeatureConfiguration.cs#L197-L203

But there may be cases where it does not apply properly.

**Comment by Soap-141** [comment]
Here is my sample [SampleDisplaySize.zip](https://github.com/unoplatform/uno/files/12577790/SampleDisplaySize.zip) @ajpinedam

**Comment by MartinZikmund** [comment]
I think we should both support the `FeatureConfiguration` properties properly, and also verify we accurately model how Windows handles text scaling based on accessibility settings:

![image](https://github.com/unoplatform/uno/assets/1075116/4a7fcb87-badd-4ad9-a078-86c92ee7129c)
